src/components/pages/Map.tsx

```diff
@@ -137,16 +143,21 @@ function Map() {
           onBoundsChanged={onMapBoundsChanged}
           zoom={13}
         >
-          {/* <Marker position={positionTokyo} />
-          <InfoWindow position={positionTokyo} options={infoWindowOptions}>
-            <div style={divStyle}>
-              <h1>ガンダムベース東京</h1>
+          
+          {/* Railsから取得したデータを、Marker地図上に表示 */}
+          {shops.map((shop: shops) => (
+            <div>
+              <Marker position={shop} />
+              <InfoWindow position={shop} options={infoWindowOptions}>
+                <div style={divStyle}>
+                  <h1>{shop.name}</h1>
+                </div>
+              </InfoWindow>
             </div>
-          </InfoWindow> */}
+          ))}
```

Comment:
Rails側のShopに緯度経度のプロパティなさそうですよね？？
とりあえず適当な座標をpositionプロップスに渡してあげたら表示はされるようになりました！

```suggestion
          {shops.map((shop, index) => (
            <>
              <Marker position={{ lat: 35 + index, lng: 139 + index }} key={index} />
              <InfoWindow position={{ lat: 35 + index, lng: 139 + index }} options={infoWindowOptions} key={index}>
                <div style={divStyle}>
                  <h1>{shop.name}</h1>
                </div>
              </InfoWindow>
            </>
          ))}
```

![スクリーンショット 2022-06-21 12 32 13](https://user-images.githubusercontent.com/18608862/174711430-40b25c70-0b2c-45ca-8209-a6bab6f1f444.png)

Reply:
なるほど！
Rails側のShopから緯度経度のプロパティが渡されていないのが原因みたいですね。